mars/tensor/fuse/jax.py

```diff
@@ -0,0 +1,45 @@
+from ...serialize import DataTypeField
```

Comment:
Apache 2 license should be prepended. Rename the file to avoid relative import behavior in Python 2.